src/experimental/react/controllers/viewport-animation-utils.js

```diff
@@ -0,0 +1,115 @@
+/* eslint max-statements: ["error", 50] */
```

Comment:
Since this is not a react component, this should go into `src/controllers` instead of `src/react/controllers`